test_e2e/expected-structured-output/discord/1099442333440802930.txt.json

```diff
@@ -1,50 +1,94 @@
 [
```

Comment:
Since we have custom logc on how we map the data into a file, should also validate the download content: [elasticsearch test](https://github.com/Unstructured-IO/unstructured-ingest/blob/main/test/integration/connectors/elasticsearch/test_elasticsearch.py#L201)

Reply:
Ok, then I will have to add a new integration test for Discord, as now it has only e2e test..